src/Utilities/StrideIterator.hh

```diff
@@ -0,0 +1,35 @@
+//---------------------------------Spheral++----------------------------------//
```

Comment:
I had to add this header as well to the CMakeLists.txt file so that it would be installed, after including PairwiseField.hh.

Reply:
Fixed this one too